condition/firmware_install.go

```diff
@@ -41,14 +45,22 @@ type FirmwareInstallTaskParameters struct {
 	FirmwareSetID uuid.UUID `json:"firmware_set_id,omitempty"`
 }
 
+func (p *FirmwareInstallTaskParameters) MustJSON() []byte {
```

Comment:
Curious where the `MustJSON` function being used?

Reply:
I'm going to use it in ConditionAPI (and likely orchestrator). But it really belongs here where it's defined.